recipes-devtools/yaml-cpp/yaml-cpp_0.5.2.bb

```diff
@@ -11,8 +11,8 @@ PR = "r0"
 S = "${WORKDIR}/yaml-cpp-release-${PV}"
 
 SRC_URI = "https://github.com/jbeder/yaml-cpp/archive/release-${PV}.tar.gz"
-SRC_URI[md5sum] = "2728af8a15e2b2c407730c45b99b274b"
-SRC_URI[sha256sum] = "6fb92f6f5925e0af918ffbb90acf19b7b88706ebcd40fc186b7caa76609b6350"
+SRC_URI[md5sum] = "96bdfa47d38711737d973b23d384d4f2"
```

Comment:
On indigo (bmwcarit/master) and kinetic-experimental-v3-alpha1, yaml-cpp is updated
to 0.5.3 and loaded with git. So, this issue of this hunk has been resolved in indigo (bmwcarit/master)
and kinetic-experimental-v3-alpha1.